providers/pool.rb

```diff
@@ -232,6 +236,38 @@ def configure
       shell_out!(cmd)
       @new_resource.updated_by_last_action(true)
     end
+    if is_new_start_mode
+      was_updated = true
+      cmd = "#{appcmd(node)} set apppool \"/apppool.name:#{@new_resource.pool_name}\""
+      cmd << " /startMode:#{@new_resource.start_mode.to_s}"
+      Chef::Log.debug(cmd)
+      shell_out!(cmd)
+      @new_resource.updated_by_last_action(true)
```

Comment:
Yeah, this was done because someone else changed all the code to do this `updated_by_last_action(true)` in a pull request so I think i just kept that logic but I agree.  This likely needs to be a separate pull request as it effects almost all of the `providers`.